polar-core/src/vm.rs

```diff
@@ -2265,7 +2260,10 @@ impl PolarVirtualMachine {
                 || {
                     let mut rule_strs = "APPLICABLE_RULES:".to_owned();
                     for rule in rules {
-                        rule_strs.push_str(&format!("\n  {}", self.rule_source(&rule)));
+                        rule_strs.push_str(&format!(
+                            "\n  {}",
+                            /*self.rule_source(&rule)*/ rule.to_polar()
```

Comment:
```suggestion
                            rule.to_polar()
```

but rustfmt will want to single line it probably

Reply:
reverted to the previous state in cf526f7